integrations/tensorflow/e2e/BUILD

```diff
@@ -57,6 +56,23 @@ package(
     ]
 ]
 
+# TODO(GH-1620): Merge this with the above after fixing the failure on GitHub
+# Actions.
+[
+    iree_py_test(
+        name = name,
+        srcs = [name + ".py"],
+        python_version = "PY3",
+        tags = ["nokokoro", "noga"],
+        deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+            "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+        ],
+    )
+    for name in [
+        "keras_lstm_static_test",
```

Comment:
Looks like you could move this to the block below

```
for name in [
        "conv_test",
        "linspace_test",
        "math_test",
    ]
```

(and we should enable those... why are they disabled? IIRC, they had Vulkan dependencies? Tracking issues would help)

Reply:
Yeah these should be possible to re-enable now that we can specify supported backends.

Reply:
Those ones are not marked as `nokokoro`... But it seems kokoro are not running TensorFlow integration tests anyway? I'm a bit confused regarding the test infra support matrix now.

Reply:
Kokoro doesn't include this directory: https://github.com/google/iree/blob/5c2ab3e535580e7a9f2a03314f9a29bd6cd80785/build_tools/bazel/build.sh#L61-L72

The test infra matrix needs some cleanup (and a clear plan :P)

Reply:
I thought I deleted the extra nokokoro tags, but must have missed these

Reply:
Okay merged and removed remaining nokokoro tags.